core-web/apps/dotcms-ui/src/app/view/components/dot-toolbar/components/dot-toolbar-user/dot-toolbar-user.component.scss

```diff
@@ -7,7 +7,7 @@
 
 p-avatar {
     cursor: pointer;
-    padding: $spacing-2 0;
+    padding: 0.83rem 0;
```

Comment:
We should use the `$spacing `variable here

Reply:
@rjvelazco Fixed thanks!